webapp/src/routes/BlockProducers/index.js

```diff
@@ -174,22 +179,22 @@ const Producers = () => {
   }, [dataHistory, loadingHistory])
 
   return (
-    <Box>
+    <Grid>
```

Comment:
if you won't be using the real potential of the `Grid` or `Box` component is much better just use a simple `div` as a wrapper